src/main/resources/reference.conf

```diff
@@ -1281,6 +1283,7 @@ resourceTypes = {
       }
     }
     reuseIds = true
```

Comment:
You _may_ want to set `reuseIds` to false, just so that an ID without an auth domain can't be re-used for something that does require one. Though, I do doubt that it's an issue, since TDR is generating new UUIDs for every new Snapshot.

Reply:
Ah yeah, might as well to be safe